openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -1584,9 +1584,9 @@ The Credential Issuer determines the number of the Credentials issued in the Cre
 Implementers should be aware that this specification uses several specifications that are not yet final specifications. Those specifications are:
 
 * OpenID Federation 1.0 draft -43 [@!OpenID.Federation]
-* SD-JWT-based Verifiable Credentials (SD-JWT VC) draft -08 [@!I-D.ietf-oauth-sd-jwt-vc]
-* Attestation-Based Client Authentication draft -05 [@!I-D.ietf-oauth-attestation-based-client-auth]
-* Token Status List draft -11 [@!I-D.ietf-oauth-status-list]
+* SD-JWT-based Verifiable Credentials (SD-JWT VC) draft -10 [@!I-D.ietf-oauth-sd-jwt-vc]
```

Comment:
I'm gonna jam out a -11 one way or the other by the time this goes in and gets published  
```suggestion
* SD-JWT-based Verifiable Credentials (SD-JWT VC) draft -11 [@!I-D.ietf-oauth-sd-jwt-vc]
```

Reply:
So it was written, so it was done https://mailarchive.ietf.org/arch/msg/oauth/-blEIowkxM-IG1oTxp1UyS406_w/

Reply:
"Apply suggestions from Brian's review " 

review? review?! that was more of a prophecy 😂

Reply:
that is the default text!